src/stdlib_experimental_quadrature.fypp

```diff
@@ -1,5 +1,6 @@
 #:include "common.fypp"
-
+#:set WEIGHT_FUNS = ["sin", "cos", "pole"]
+#:set QUAD_OK = False
```

Comment:
How is this variable managed? Through the CMakefile? Manually?
Is it just to avoid the issue with GFortran 9.2?

Reply:
I tested it with `QUAD_OK = True` and ifort 19.1, and it went fine.

Reply:
Ack, all this stuff related to "quad" and such is stuff I'm playing with on the side and should not be in this PR. I will remove it.